src/main/java/org/prebid/server/auction/TargetingKeywordsCreator.java

```diff
@@ -177,6 +180,12 @@ boolean isNonZeroCpm(BigDecimal price) {
         if (StringUtils.isNotBlank(vastCacheId)) {
             keywordMap.put(HB_VAST_ID_KEY, vastCacheId);
         }
+        if (StringUtils.isNotBlank(vastCacheId) || StringUtils.isNotBlank(cacheId)) {
+            if (StringUtils.isNotBlank(cacheHost)) {
```

Comment:
1. No need this condition anymore, we'll always put `HB_CACHE_*` targeting.
2. Add `HB_CACHE_HOSTPATH` key.